src/runtime.rs

```diff
@@ -130,7 +131,17 @@ impl FutureIntoCoroutine {
             // Here we have left the trio thread and are inside a thread provided by the Tokio runtime
 
             // Actual run of the Tokio future
-            let ret = fut.await;
+            let ret = std::panic::AssertUnwindSafe(fut).catch_unwind().await;
+            let ret = ret.unwrap_or_else(|panic_err| {
+                let msg = match panic_err.downcast::<&str>() {
+                    Ok(msg) => (*msg).to_owned(),
```

Comment:
```suggestion
                    Ok(msg) => msg.to_string(),
```

I'm a `to_string` guy

Reply:
👍